.github/ISSUE_TEMPLATE/feature-request.md

```diff
@@ -7,6 +7,10 @@ labels: sig/cluster-management, kind/feature
 
 <!-- Please use this template if you want to submit a new feature request.-->
 
-**What feature would you like to be added?**
+### User Story
```

Comment:
I'd like that we keep the question headers, they're especially useful for external contributors.

Reply:
Okay, let's keep the explanation there, and what about also helping issuers creating a proper user story with a template:
```suggestion
### Description of the feature you would like to add / User Story
<!-- Please, try to write your user story in a format like the one has below:
As a <user persona>
I would like to <functionality>
in order to <benefit>
-->
```